media-gfx/gscan2pdf/gscan2pdf-2.1.7.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils perl-module
+
+DESCRIPTION="Scan documents, perform OCR, produce PDFs and DjVus"
+HOMEPAGE="http://gscan2pdf.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# OCR tests fail with tesseract[opencl], not fixed by addpredict
+RESTRICT="test"
+
+RDEPEND="
+	dev-perl/Config-General
+	dev-perl/Date-Calc
+	dev-perl/Data-UUID
+	dev-perl/Filesys-Df
+	dev-perl/glib-perl
+	dev-perl/GooCanvas2
+	dev-perl/Gtk3
+	dev-perl/Gtk3-SimpleList
+	dev-perl/HTML-Parser
+	dev-perl/Image-Sane
+	dev-perl/Locale-gettext
+	dev-perl/List-MoreUtils
```

Comment:
Please sort this list in alphabetical order. Li < Lo, and media/ < virtual/

Reply:
I'll sort the two dev-perl's but the virtuals are deliberately immediately below the other Perl modules.

Reply:
(but media-gfx/* dependencies above virtuals?)